api/v1alpha1/bucket_types.go

```diff
@@ -21,7 +21,10 @@ import (
 )
 
 const (
-	BucketFinalizer = "bucket.objectstorage.giantswarm.io"
+	BucketFinalizer     = "bucket.objectstorage.giantswarm.io"
+	AzureSecFinalizer   = "giantswarm.io/object-storage-operator"
```

Comment:
Can we call this AzureSecretFinalizer? Also the value should the similar to `bucket.objectstorage.giantswarm.io` so maybe `secret.objectstorage.giantswarm.io`

Reply:
I can't call it `AzureSecretFinalizer` because of linter: 
https://app.circleci.com/pipelines/github/giantswarm/object-storage-operator/642/workflows/1ac75edb-f928-4397-b084-ff1cbd1bec12/jobs/2582

Reply:
you can add a comment to ignore this warning for this line :)